ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3129,6 +3129,10 @@ literals:
     fixed_min: 0.0
     flexible_max: 2000.0
 
+- high_water_temperature_setpoint:
```

Comment:
This is not descriptive enough; are you trying to model the threshold that will trigger an alarm or the control setpoint needed for the system?

Reply:
yes, trying to model the threshold that will trigger the alarm

Reply:
Ok thanks for clarifying. Typically if you include the alarm point itself there is no need to model the threshold that triggers the alarm; I would suggest removing this field from your model.